src/pytest_inmanta_lsm/plugin.py

```diff
@@ -234,37 +232,50 @@ def remote_orchestrator_shared(
                     "The module being tested is a v1 module but it is also installed as a v2 module. Local compiles will use"
                     "the v2, but only the v1 will by synced to the server."
```

Comment:
This is not necessarily true anymore: if the v2 is installed in editable mode it will be synced to the server.

Reply:
Indeed, fixed